README.md

```diff
@@ -151,11 +151,9 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
 
 ### Using `npm`:
 
-1. If you don't already have npm on your machine, install it. On OSX with [Homebrew](https://brew.sh/) that looks like:
+1. If you don't already have npm on your machine, install it using npm's recommended method: [a version manager](https://www.npmjs.com/get-npm).
+    If you have already installed npm with `brew` or `apt`, it is possible you will run into an `EACCES` error (related to how many system packagers install node) while installing wrangler. You can either uninstall that version of node and use the npm recommended install method, or use `cargo` instead.
 
-    ```
-    $ brew install nodejs
-    ```
     
 1. Install Wrangler by running:
```

Comment:
the numbering gets weird here; should be corrected somehow. likely would be corrected by making the bit about system package installers into a footnote

Reply:
Github still renders it fine, although the markdown numbers don't increment, the rendered output does. Will move the bit about system installers "inside" point #1 like the openssl one is.

Reply:
i can't be the only one who opens readmes in my text editor locally

Reply:
Should be a separate PR as the entire doc is written with this numbering